client/src/components/application/NonlinearStepper/GeneralInfo.js

```diff
@@ -15,6 +21,55 @@ const GereralInfos = ({ handleFormInput, formValues, errorMsg }) => {
     'Khan Younis',
     'Rafah',
   ];
+  const GazaUniversity = [
+    "I didn't go to university",
+    'Islamic University of Gaza',
+    'Al Azhar University',
+    'Palestine University',
+    'UCAS',
+    'Al Quds Open University',
+    'University College of Science and Technology',
+    'Al Aqsa University',
+    'Gaza University',
+    'I went to university outside of Gaza',
+  ];
+
+  const KhalilUniversity = [
+    "I didn't go to university",
+    'Palestine Polytechnic University (PPU)',
+    'Hebron University',
+    'Bethlehem University',
+    'Birzeit University',
+    'Al Najah University',
+    'Arab American University',
+    'Al Quds Open University',
+    'Al Quds University',
+    'Palestine Technical University - Kadoori',
+    'IT College - Vocational Center',
+    'University outside of Palestine',
+    'Other',
+  ];
+  const advert = [
+    'Gaza Sky Geeks Social Media (Facebook, Twitter, Instagram)',
+    'Friend or family member who is a Code Academy graduate',
+    'I came to a GSG workshop or event (such as “Coding for Everyone”)',
+    'Online / browsing the web',
+    'Jobs.ps',
+    'Other',
+  ];
+  const codeExperience = [
+    'No Experience',
+    'Some experience as hobby',
+    'A lot of experience as a hobby',
+    'I have a university degree in Computer Science or another related field',
+    'I have had a paid job as a developer.',
+  ];
+  const primaryMotivationForm = [
+    'I want to gain employment/earn income as a developer.',
+    "I want to learn something new - I don't know if I want a coding career",
+    'I want to be part of the GSG and Code Academy community',
+  ];
+
   const KhalilAreas = ['Old Town', 'New Town'];
```

Comment:
can you please transfer all constant in a file beside index file and just call it here.